__tests__/__snapshots__/Send.Portrait.snapshot.tsx.snap

```diff
@@ -112,6 +112,7 @@ exports[`Component Send - test Send Landscape - snapshot 1`] = `
                       "marginLeft": 5,
                     }
                   }
+                  testID="send.addressplaceholder"
```

Comment:
Can't we remove tests that are against the display mode we're not supporting?

Reply:
I think I need more info. How I can activate `the mode that you are talking about` in zingo-mobile????? Exits something in `zingolib` that can activate `that mode`, maybe display-mode????? I'm not familiar with the different modes....

Reply:
sorry, I could not understand this properly.